### PR TITLE
Added a HighContrast variant for tags

### LINKED ISSUE #12647: Bug report: color accessibility with tags
### Describe the bug

Some users have expressed difficulty reading tags on https://ethereum.org/en/wallets/find-wallet/ 

This is an issue both on light mode as well as dark mode. The problem is that color contrast between the tag background and tag text is too low. Even though it follows accessibility standards (>4.5) the perceived readability is low and we should increase the contrast ratio.

### To reproduce

1. Go to https://ethereum.org/en/wallets/find-wallet/
2. See error in wallet listing (tags)

Screenshot 2024-04-03 at 12 13 44

Screenshot 2024-04-03 at 12 13 40

### Expected behavior

increase contrast by changing background color or text color (or both)

### Screenshots

_No response_

### Desktop (please complete the following information)

_No response_

### Smartphone (please complete the following information)

_No response_

### Additional context

_No response_

### Would you like to work on this issue?

- [ ] Yes
- [X] No

### PR DESCRIPTION
**Pull request by Dharmik79**
## Description

Added a new variant for highContrast for the tag in the find Wallet Page. [Fixes #12647 ]

### CODE REVIEW

**Comment by nloureiro** [comment]
@Dharmik79 great work, thank you

small adjustment > The tags should not have a border
Screen Shot 2024-04-23 11 09 44 AM

Thank you!

**Comment by konopkja** [comment]
Screenshot 2024-04-23 at 16 35 55
i see border in the preview deploy for the tags, that should not be there.

**Comment by Dharmik79** [comment]
@nloureiro I have removed borders from the tags.

File: src/@chakra-ui/components/Tag/index.ts [hunk 1]
```diff
@@ -112,10 +112,26 @@ const variantOutline = definePartsStyle((props) => {
...
+  const defaultStyles = tagTheme.variants?.outline(props)
+  const statusStyles = getStatusStyles(status, "highContrast")
+  return {
+    container: {
+      ...defaultStyles?.container,
+      boxShadow: "none",
+
+      // Remove default dark mode styles
+      _dark: {},
+      ...statusStyles.container,
```
Comment (wackerow) [inline src/@chakra-ui/components/Tag/index.ts:126]: Is this correct?

Light:
image

Dark:
image

For the background I see `body.light` in light mode, and `body.base` in dark

Comment (nloureiro) [reply src/@chakra-ui/components/Tag/index.ts:126]: Screen Shot 2024-04-24 09 04 49 AM

it should be body.light on both

Comment (wackerow) [reply src/@chakra-ui/components/Tag/index.ts:126]: Hm... my comment doesn't seem to align with my screenshots as I look back at this... thanks, can disregard

**Review by wackerow — APPROVED** [review]
image

Looks good, thanks @Dharmik79!

**Pull request merged**